apps/web/src/App.tsx

```diff
@@ -205,7 +208,11 @@ function App() {
                   <Route path={ROUTES.QUICK_START_SETUP_SUCCESS} element={<InAppSuccess />} />
                   <Route path={ROUTES.ACTIVITIES} element={<ActivitiesPage />} />
                   <Route path={ROUTES.SETTINGS} element={<SettingsPage />} />
-                  <Route path={ROUTES.INTEGRATIONS} element={<IntegrationsListPage />} />
+                  <Route path={ROUTES.INTEGRATIONS} element={<IntegrationsListPage />}>
+                    <Route path="create" element={<SelectProviderSidebar />} />
+                    <Route path="create/:channel/:providerId" element={<CreateProviderPage />} />
+                    <Route path=":integrationId" element={<UpdateProviderPage />} />
```

Comment:
can we reuse the ROUTES here, if those are nested /integrations maybe we can create a new dedicated enum ROUTES_INTEGRATION or something like this. wdyt?